bin/triggers_check

```diff
@@ -85,7 +85,7 @@ module AutoHCK
     cli = CLI.new
     cli.parse(ARGV)
 
-    @logger = MonoLogger.new($stdout)
+    @logger = MonoLogger.new($stderr)
     @logger.level = cli.debug ? 'DEBUG' : 'INFO'
```

Comment:
The same logger is used at the end to write: `Some drivers are changed, running tests` or `Any drivers aren't changed, not running tests`. Do you think this log is an actual product of the operation or an actual product is it the exit code?

Reply:
It is the exit code in this case. We don't expect anybody to parse that log.